src/cpp/src/perf_metrics.cpp

```diff
@@ -101,7 +101,7 @@ void PerfMetrics::evaluate_statistics(std::optional<TimePoint> start_time) {
 
         auto ttft = tok_times[0] - start_time_val;
         raw_metrics.m_times_to_first_token = std::vector<MicroSeconds>();
-        raw_metrics.m_times_to_first_token.emplace_back(ttft / batch_sizes[0]);
+        raw_metrics.m_times_to_first_token.emplace_back(ttft);
```

Comment:
If we have batch 10 and it takes 1 sec until the first token is generates then time to the first token is still 1 sec not 100 ms! Therefore i removed `/ batch_sizes[0]`

Reply:
I agree, but should we notify llm_bench about the change?

Reply:
I asked @eaidova what she thinks. Let's wait her answer

Reply:
@peterchen-intel  @wgzintel  do you have objections? What the reason to divide first token latency on batch?

Reply:
As i see from blame, it was added by @ialbrecht.